src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-junitChkBx.html

```diff
@@ -0,0 +1,5 @@
+<div>
+<br>
+Generate a JUnit style test report, named <b> junittestresults.xml, </b> to be stored in the<b> matlabTestArtifacts</b> folder of the current Jenkins workspace.
```

Comment:
Remove extraneous spaces around `junittestresults.xml,`

`named <b> junittestresults.xml, </b> to` -> `named <b>junittestresults.xml</b>, to`

Reply:
done